src/styles/blocks/link-button.scss

```diff
@@ -0,0 +1,27 @@
+.link-button {
+  display: flex;
+  justify-content: center;
+  padding: 15px 0;
+
+  font-weight: 700;
```

Comment:
You may consider adding a 'font-size' and 'line-height' properties to style the font consistently across devices.